rewrite-core/src/test/java/org/openrewrite/text/ChangeTextTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.text;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.test.SourceSpecs.text;
+
+class ChangeTextTest implements RewriteTest {
+    @Test
+    void changeText() {
+        rewriteRun(
+          spec -> spec.recipe(new ChangeText("after")),
+          text("before", "after"),
+          text("", "after")
+        );
+    }
```

Comment:
I don't think we need this test. `ChangeText` is very simple and already exercised in plenty of tests

Reply:
removed.